src/plugins/intel_gpu/src/graph/impls/ocl/scaled_dot_product_attention.cpp

```diff
@@ -233,7 +270,7 @@ struct scaled_dot_product_attention_impl : multi_stage_primitive<scaled_dot_prod
     }
 
 public:
-    static kernel_params_t get_kernel_params(const kernel_impl_params& impl_param, bool is_dynamic, bool indirect = false) {
+    static kernel_params_t get_kernel_params(const kernel_impl_params& impl_param, bool is_dynamic, bool indirect = false, bool force_sdpa_opt = false) {
```

Comment:
It seems `force_sdpa_opt` can be removed from this function, as it's being manually set in the create() function